test/unified_format.py

```diff
@@ -624,6 +625,14 @@ def get_lsid_for_session(self, session_name):
             # session has been closed.
             return self._session_lsids[session_name]
 
+    def advance_cluster_times(self) -> None:
+        """Manually synchronize entities when desired"""
+        if not self._cluster_time:
+            self._cluster_time = self.test.client.admin.command("ping").get("$clusterTime")
+        for entity in self._entities:
```

Comment:
```suggestion
        for entity in self._entities.values():
```

The current version iterates over the names instead of the values and never passes the `isinstance` check.